docs/tutorials/handler.md

```diff
@@ -0,0 +1,389 @@
+
+Rugs are under the `.atomist` directory.  The `manifest.yml` file
+contains metadata about the Rug project including the archive name,
+group, version, and dependency on the Rug runtime.  There are
+different subdirectories for editors, handlers, and tests.  This
+starter Rug project has a sample editor, command handler, event
+handler, and tests for each.  It's a good idea at this point to edit
+the manifest file, changing the group value to be the name of your
+GitHub organization.
+
+## Install the Rug CLI
+
+When developing Rugs, the Rug CLI enables you to compile, test, and
+publish your Rugs locally.  So before we starting writing Rugs, you should
+[install the Rug CLI][cli-install].
+
+[cli-install]: /user-guide/interfaces/cli/install.md (Rug CLI Install)
+
+Now it's time to write your new handler.
+
+## Write your handler
+
+Rather than editing the sample event handler provided in the starter
+Rug project, let's avoid any copy/edit errors by using an editor to
+add our event handler.  Since we've already run Rugs in Slack and we
+just installed the Rug CLI, let's see what running an editor in the
+Rug CLI looks like.  We will add an event handler called
+"CloseIssueThanks" with the following command.
+
+```console
+$ rug edit atomist-rugs:rug-editors:AddTypeScriptEventHandler \
+    handlerName=CloseIssueThanks \
+    description="simple event handler that sends a thank you message" \
+    pathExpression="/Issue()"
+```
+
+This editor will create files for your event handler and its test.
+The path expression we provide is the piece of information that
+matches incoming events to event handlers.  In this example, we are
+matching against all events having to do with issues.
+
+Rugs are most often written in [TypeScript][ts], which is an superset
+of JavaScript that introduces strong typing and transpiles into
+JavaScript.  To fully benefit from the TypeScript development process,
+you should install [Node.js][node], which also installs the `npm`
+utility.  Once `npm` is available on your system, you can install
+TypeScript tools and the [yarn][] package manager with the following
+command.
+
+```console
+$ npm install -g yarn typescript tslint
+```
+
+Once you have those tools installed, change into the `.atomist`
+directory of your Rug project and install the project's TypeScript
+dependencies using yarn.
+
+```console
+$ cd .atomist
+$ yarn install
+```
+
+<div class="ss-container">
+  <img src="../images/yarn-install.png" alt="Install TypeScript Dependencies with yarn" class="ss-small">
+</div>
+
+[ts]: https://www.typescriptlang.org/ (TypeScript)
+[node]: https://nodejs.org/ (Node.js)
+[yarn]: https://yarnpkg.com/en/ (Yarn Package Manager)
+
+!!! tip "TypeScript Development"
+    If you do not have a TypeScript development environment set up, we
+    recommend using [Visual Studio Code][vs-code].
+
+[vs-code]: https://code.visualstudio.com/ (Visual Studio Code)
+
+With our tooling and dependencies installed, we can begin developing
```

Comment:
throughout the document we switch between referring to `our` and `your`. I think we should choose and stick with one.